src/main/java/fr/insee/pearljam/api/controller/InterviewerController.java

```diff
@@ -74,42 +75,122 @@ public ResponseEntity<String> postInterviewers(HttpServletRequest request, @Requ
 	 *         or {@link HttpStatus} FORBIDDEN
 	 */
 	@ApiOperation(value = "Get interviewers")
-	@GetMapping(path = "/interviewers")
+	@GetMapping(path = Constants.API_INTERVIEWERS)
 	public ResponseEntity<Set<InterviewerDto>> getListInterviewers(HttpServletRequest request) {
 		String userId = utilsService.getUserId(request);
```

Comment:
Don't you think the user id should be processed by the security stack ? And maybe you could retrieve it with Principal/Authentication object as arg to the method.